content/support-matrix/delfin/add-driver-support/_index.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Add new driver support matrix
+description: "Document to add new delfin driver support matrix"
+weight: 20
+disableToc: false
+tags: ["support matrix", "delfin", "drivers"]
+---
+
+Find below steps to add new delfin driver support matrix
```

Comment:
Give link to sample template in the begining

Reply:
Added sample template in same page as per our discussion